### PR TITLE
Fix: healthcheck wait

### PR DESCRIPTION
**Pull request by PeterSmallenbroek**
## Describe your changes

Brief description of your changes - not in-depth because the bulk of the description should be in the task on DevOps.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] I have added tests wherever relevant
- [ ] I have made sure that all automated checks pass before requesting a review
- [ ] I do not need any deviation from our PR guidelines

### CODE REVIEW

File: .github/workflows/test_e2e.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Run Services with Docker
         working-directory: ./services
         run: |
-          docker --log-level 'warn' compose -f docker-compose.yml up -d --quiet-pull
+          docker --log-level 'warn' compose -f docker-compose.yml up -d --quiet-pull --wait --wait-timeout 300
```
Comment (elwinschmitz) [inline .github/workflows/test_e2e.yml:45]: Is this a recent addition to Docker to "wait untill all healthchecks are ready"?? Awesome. :)

Comment (PeterSmallenbroek) [reply .github/workflows/test_e2e.yml:45]: Yes, at least that's what I'm interpreting from: https://stackoverflow.com/a/76422970
But, now the struggle is within the healthcheck itself to get it to return 'Healthy'

File: services/121-service/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,7 @@
 FROM node:18-slim as base
 
+RUN apt-get update && apt-get install curl -y
```
Comment (aberonni) [inline services/121-service/Dockerfile:3]:
```suggestion
# Used by healthchecks
RUN apt-get update && apt-get install curl -y
```

File: services/mock-service/Dockerfile [hunk 3]
```diff
@@ -1,5 +1,7 @@
 FROM node:18-slim as base
 
+RUN apt-get update && apt-get install curl -y
```
Comment (aberonni) [inline services/mock-service/Dockerfile:3]:
```suggestion
# Used by healthchecks
RUN apt-get update && apt-get install curl -y
```

**Review by aberonni — DISMISSED** [review]

**Review by aberonni — APPROVED** [review]

**Pull request merged**